portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -21,29 +21,54 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.*;
+import com.google.sps.data.Comment;
+import com.google.gson.Gson;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private final ArrayList<String> comments = new ArrayList<String>();
-  
+    private static final String contentParameter = "content";
+    private static final String timestampParameter = "timestamp";
```

Comment:
Whoops, sorry I gave you incorrect instructions earlier. For constants, name them snake case, ie. `CONTENT_PARAMETER` instead.